.env.development

```diff
@@ -6,7 +6,7 @@ CREDENTIALS_BASE_URL='http://localhost:18150'
 CSRF_TOKEN_API_PATH='/csrf/api/v1/token'
 ECOMMERCE_BASE_URL='http://localhost:18130'
 LANGUAGE_PREFERENCE_COOKIE_NAME='openedx-language-preference'
-LMS_BASE_URL='http://localhost:18000'
+LMS_BASE_URL='http://local.openedx.io:8000'
```

Comment:
[question] I'm not sure that we need to have changes in `.env` files with changing `localhost` to `local.openedx.io`.

Reply:
This had to be done locally because during development I use my tutor plugin to add a new MFE, like:

```
from tutormfe.hooks import MFE_APPS

@MFE_APPS.add()
def _add_my_mfe(mfes):
mfes["catalog"] = {
"repository": "https://github.com/openedx/frontend-app-catalog.git",
"port": 1998,
"version": "master", # optional, will default to the Open edX current tag.
}
return mfes
```

I'm removing this changes from this PR because in production we are using the updated fork of the tutor-mfe for successful deployment.